### DISCUSSION TITLE
npm install not working

### DISCUSSION THREAD

**Discussion by Danko1993** [styled-components/styled-components#4039] [Q&A] [answered]
When i try to install styled-components by npm install styled-components
i receive an error
npm ERR! Cannot read properties of null (reading 'edgesOut')
npm ERR! A complete log of this run can be found in: C:\Users\Daniel\AppData\Local\npm-cache\_logs\2023-06-13T22_22_15_607Z-debug-0.log (file attached)
[2023-06-13T22_22_15_607Z-debug-0.log](https://github.com/styled-components/styled-components/files/11739951/2023-06-13T22_22_15_607Z-debug-0.log)
using node v20.2.0
Any idea how to solve this?
i tried to re-install npm, clear npm's cache deleting node modules and pckage-lock,json also didnt helped.

**Comment by letFlavinho** [comment]
also happenned to me and a someone that i know, we did the npm install styled-components@latest, it works.

**Comment by quantizor** [comment, marked as answer]
This was an npm bug, not a styled-components issue. The error about `edgesOut` was a known npm issue in certain versions. Upgrading npm or using `npm install styled-components@latest` (as confirmed in the thread) resolves it.
